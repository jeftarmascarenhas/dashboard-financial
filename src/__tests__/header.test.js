import React from 'react'
import { render, cleanup } from '@testing-library/react'

import Header from 'components/header'

afterAll(cleanup)

describe('<Header />', () => {
  it('rendering', () => {
    const { asFragment } = render(<Header />)
    expect(asFragment()).toMatchSnapshot()
  })
})
