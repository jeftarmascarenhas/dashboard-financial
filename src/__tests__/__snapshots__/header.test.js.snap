// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> rendering 1`] = `
<DocumentFragment>
  .c3 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  border-bottom: 1px solid #2f3779;
  height: 6.375rem;
}

.c2 {
  color: #7f84b9;
  width: 28px;
  height: 28px;
}

.c1 {
  border: 0;
  border-bottom: 1px solid #2f3779;
  border-right: 1px solid #2f3779;
  box-shadow: none;
  outline: none;
  width: 6.375rem;
  height: 6.375rem;
  cursor: pointer;
}

.c1 > svg {
  -webkit-transition: all 0.20s ease-in-out;
  transition: all 0.20s ease-in-out;
}

.c1:hover svg {
  color: #fff;
}

.c4 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 6.375rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding-left: 1rem;
}

.c7 {
  background: rgba(0,0,0,0.82);
  position: absolute;
  display: none;
  top: 6.375rem;
  left: 0;
  right: 0;
  bottom: 0;
  width: 100%;
  min-height: 100vh;
  padding: 0 1rem;
  padding-bottom: 2rem;
  overflow-y: hidden;
  z-index: 10;
}

.c7.active {
  display: block;
}

.c6 {
  border: 0;
  box-shadow: none;
  outline: 0;
  display: block;
  width: 100%;
  padding: 0.3rem;
  color: #fff;
  height: 100%;
  background: transparent;
  font-size: 1.4rem;
}

.c6::-webkit-input-placeholder {
  color: rgba(47,55,121,0.9);
}

.c6::-moz-placeholder {
  color: rgba(47,55,121,0.9);
}

.c6:-ms-input-placeholder {
  color: rgba(47,55,121,0.9);
}

.c6::placeholder {
  color: rgba(47,55,121,0.9);
}

.c5 {
  color: #fff;
  width: 25px;
  height: 25px;
  cursor: pointer;
}

.c10 {
  background: #181757;
  padding: 1rem 2rem;
  margin-bottom: 1.5rem;
  display: block;
  color: #fff;
  min-height: 6rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c9 {
  color: #fff;
  width: 28px;
  height: 28px;
  cursor: pointer;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 1rem;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
}

<header
    class="c0"
    data-testid="app-header"
  >
    <button
      class="c1"
    >
      <svg
        aria-hidden="true"
        class="c2 c3"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M4 6h16v2H4zm4 5h12v2H8zm5 5h7v2h-7z"
        />
      </svg>
    </button>
    <div
      class="c4"
    >
      <svg
        aria-hidden="true"
        class="c5 c3"
        fill="currentColor"
        focusable="false"
        viewBox="0 0 50 50"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M23 36c-7.2 0-13-5.8-13-13s5.8-13 13-13 13 5.8 13 13-5.8 13-13 13zm0-24c-6.1 0-11 4.9-11 11s4.9 11 11 11 11-4.9 11-11-4.9-11-11-11z"
        />
        <path
          d="M32.682 31.267l8.98 8.98-1.414 1.414-8.98-8.98z"
        />
      </svg>
      <label
        class="sr-only"
        for="searchInput"
      >
        Search Symbol
      </label>
      <input
        autocomplete="off"
        class="c6"
        id="searchInput"
        name=""
        placeholder="Ex: ITSA4"
        value=""
      />
      <div
        class="c7"
      >
        <div
          class="c8"
        >
          <svg
            aria-hidden="true"
            class="c9 c3"
            fill="currentColor"
            focusable="false"
            viewBox="0 0 50 50"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M37.304 11.282l1.414 1.414-26.022 26.02-1.414-1.413z"
            />
            <path
              d="M12.696 11.282l26.022 26.02-1.414 1.415-26.022-26.02z"
            />
          </svg>
        </div>
        <div
          class="c10"
        >
          Empty List
        </div>
      </div>
    </div>
  </header>
</DocumentFragment>
`;
